lib/hanami/action/glue.rb

```diff
@@ -59,7 +59,7 @@ def finish
       #
       # @since 0.4.3
       def sending_file?
-        @_body.is_a?(::Rack::File)
+        @_body.is_a?(::Rack::File) || @_body&.first.is_a?(::Rack::File)
```

Comment:
I think we can remove the first part of the conditional because rack2 wrap all `Rack::File` instances to array like a normal body